stop_and_wait_plugin/src/stop_and_wait_plugin.cpp

```diff
@@ -202,7 +202,7 @@ std::vector<PointSpeedPair> StopandWait::maneuvers_to_points(const std::vector<c
 }
 
 std::vector<carma_planning_msgs::msg::TrajectoryPlanPoint> StopandWait::trajectory_from_points_times_orientations(
-    const std::vector<lanelet::BasicPoint2d>& points, const std::vector<double>& times, const std::vector<double>& yaws,
+    const std::vector<lanelet::BasicPoint2d>& points, std::vector<double>& times, const std::vector<double>& yaws,
```

Comment:
Better than having these methods modify their inputs on invalid conditions, throw an exception and have the caller correct the inputs right?

Reply:
This node is the main source of the trajectory. So this is the source place to modify and correct the inputs. I originally had it outside, but @adamlm suggested that from the name `trajectory_from_points_times_orientations`, this is the correct place to handle the wrong time and create correct trajectory

Reply:
After Adam's comment, I also reverted this change as we don't need to